addons/mail/static/tests/discuss/core/new_message_separator.test.js

```diff
@@ -0,0 +1,222 @@
+import { presenceService } from "@bus/services/presence_service";
+import { rpcWithEnv } from "@mail/utils/common/misc";
+import {
+    assertSteps,
+    click,
+    contains,
+    defineMailModels,
+    insertText,
+    onRpcBefore,
+    openDiscuss,
+    openFormView,
+    start,
+    startServer,
+    step,
+    triggerHotkey,
+} from "@mail/../tests/mail_test_helpers";
+import { describe, test } from "@odoo/hoot";
+import { withUser } from "@web/../tests/_framework/mock_server/mock_server";
+import { Command, mockService, serverState } from "@web/../tests/web_test_helpers";
+import { mockDate } from "@odoo/hoot-mock";
+
+/** @type {ReturnType<import("@mail/utils/common/misc").rpcWithEnv>} */
+let rpc;
+describe.current.tags("desktop");
+defineMailModels();
+
+test("remove after going to the thread twice", async () => {
+    const pyEnv = await startServer();
+    const partnerId = pyEnv["res.partner"].create({ name: "Foreigner partner" });
+    const channelId = pyEnv["discuss.channel"].create({
+        name: "test",
+        channel_member_ids: [
+            Command.create({ partner_id: partnerId }),
+            Command.create({ partner_id: serverState.partnerId }),
+        ],
+    });
+    pyEnv["mail.message"].create({
+        author_id: partnerId,
+        body: "hello",
+        message_type: "comment",
+        model: "discuss.channel",
+        res_id: channelId,
+    });
+    const env = await start();
+    rpc = rpcWithEnv(env);
+    await openDiscuss(channelId);
+    await contains(".o-mail-Thread");
+    await contains(".o-mail-Message", { text: "hello" });
+    await contains(".o-mail-Thread-newMessage hr + span", { text: "New messages" });
+    await click(".o-mail-DiscussSidebar-item", { text: "History" });
+    await contains(".o-mail-Discuss-threadName", { value: "History" });
+    await click(".o-mail-DiscussSidebar-item", { text: "test" });
+    await contains(".o-mail-Discuss-threadName", { value: "test" });
+    await contains(".o-mail-Thread-newMessage hr + span", { count: 0, text: "New messages" });
+});
+
+test("show after receiving message out of odoo focus", async () => {
```

Comment:
Would clarify with "show **_on_** receiving **_new_** message **_when_** out of odoo focus"